### PR TITLE
Add custom CRD clientset

### PR DESCRIPTION
**Pull request by varungup90**
Address https://github.com/aibrix/aibrix/issues/98

### CODE REVIEW

File: hack/verify-codegen.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+#
+cd ${SCRIPT_ROOT}
+${CODEGEN_PKG}/generate-groups.sh all \
+ github.com/aibrix/aibrix/pkg/client github.com/aibrix/aibrix/api \
+ "model:v1alpha1 autoscaling:v1alpha1" \
+ --output-base "${TEMP_DIR}" \
+ --go-header-file hack/boilerplate.go.txt
+
+# Copy everything back.
+cp -a "${TEMP_DIR}/${ROOT_PKG}/." "${SCRIPT_ROOT}/"
```
Comment (Jeffwan) [inline hack/verify-codegen.sh:53]: Let's ignore the `pkg/.DS_Store`

File: hack/verify-codegen.sh [hunk 2]
```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```
Comment (Jeffwan) [inline hack/verify-codegen.sh:1]: Let's create a follow up PR to verify the code-gen in the CI. it should blocks the Pr if there're outdated version

Comment (varungup90) [reply hack/verify-codegen.sh:1]: Sure.

File: hack/verify-codegen.sh (hunk 1 above)
Comment (varungup90) [reply hack/verify-codegen.sh:53]: Done.

**Pull request merged**